### DISCUSSION TITLE
Nach kurzer Zeit fragt SC schon wieder nach Weg-Oberflächen

### DISCUSSION THREAD

**Discussion by biketeur** [streetcomplete/StreetComplete#3574] [Feedback]
Ich hatte im Laufe des letzten halben Jahres über 50 Friedhöfe in meinem Heimatort (Großstadt) gemappt.
Jetzt muss ich feststellen, dass bei vielen von ihnen schon wieder nach den Oberflächen der Wege gefragt wird.
Vereinzelte Nachforschungen mit Vespucci haben ergeben, dass in der Chronik keine Spuren mehr von meinem mapping der Oberflächen zu finden sind. Wer entfernt diese? SC?  Und warum? Muss ich mir einen neuen Namen zulegen: Sisyphus ?

**Comment by matkoniecz** [comment]
Are you interested in answers in English based on automatic translation?

Can you link any example of such cemetery?

Which program you used for surface mapping? SC? Vespucci? JOSM?

SC is definitely not blanking `surface` tags, though for old surface data it may ask again as resurvey.

**Comment by biketeur** [comment]
Am 08.12.2021 08:35 schrieb Mateusz Konieczny ***@***.***>:
Are you interested in answers in English based on automatic translation?Yes, i am interested
Can you link any example of such cemetery?I don't know, how to create such an link. Can you explain it to me? Which SW i will need? Otherweise i can give you an adress.
Which program you used for surface mapping? SC? Vespucci? JOSM?SC
SC is definitely not blanking surface tags, though for old surface data it may ask again as resurvey.

**Reply by matkoniecz** [reply]
You can open https://www.openstreetmap.org/ and navigate to the given area and copy website URL - it is automatically updating itself and includes location

> > Which program you used for surface mapping? SC? Vespucci? JOSM?

> SC

Just to check: are you logged in? And uploading answers?

SC will ask you to log in if it will not be done and answers are uploaded by default - but it is possible to prevent SC from editing.

Are you willing to share your OSM account name? This may make things easier to check.

**Comment by biketeur** [comment]
Am 08.12.2021 09:41 schrieb Mateusz Konieczny ***@***.***>:
You can open https://www.openstreetmap.org/ and navigate to the given area and copy website URL - it is automatically updating itself and includes locationhttps://www.openstreetmap.org/#map=17/51.53016/7.61882In this cemetry I mapped about 95% of the surfaces. The chronik says, that some ways we're created by me. This is because I splitted long ways in smaller pieces. But I mapped after splitting the surfaces. I left a lot of notices because of new/changed ways. This notices were edited by the mapper Bienson (creating new ways). This new ways has still no mapped surfaces.


Which program you used for surface mapping? SC? Vespucci? JOSM?



SC

Just to check: are you logged in? And uploading answers?Of course i am logged in, and my answers will be always uploaded.
SC will ask you to log in if it will not be done and answers are uploaded by default - but it is possible to prevent SC from editing.
Are you willing to share your OSM account name? This may make things easier to check.My Accountname is biketeur

**Comment by matkoniecz** [comment]
It seems that you solved quests for ways existing for a long time such as https://www.openstreetmap.org/way/382676337/history (about month ago) and also about month ago someone else mapped some more minor paths such as https://www.openstreetmap.org/way/996336355/history and https://www.openstreetmap.org/way/996323302/history

It seems that they mapped just hours after your visit, maybe they were reviewing edits being made in their area and noticed missing paths?

See http://overpass-turbo.eu/s/1dWF Overpass Query that shows paths with surface
![screen](https://user-images.githubusercontent.com/899988/145191879-f2425780-e741-4031-962a-79d8c5a900a6.png)
